generators/server/templates/src/main/java/package/domain/_Authority.java

```diff
@@ -29,6 +29,13 @@
     @Column(length = 50)<% } %>
     private String name;
 
+    public Authority() {
```

Comment:
is this constructor needed ?
As it created a bug in our Sonar Analyze so we are now BAA. I see it after your tweet @jdubois sorry :-(